src/stories/Filter/components/ContextSwitchMenu/contextSwitchMenuStyles.ts

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled';
+import Button from 'src/Button/Button';
+
+export const ButtonLabel = styled.div({
+    fontFamily: "'Source Sans Pro', sans-serif",
+    fontSize: 16,
+    fontWeight: 700,
+    margin: 0,
+    display: 'flex',
+    alignItems: 'center',
+    textTransform: 'uppercase',
+    borderBottom: 0,
+    padding: 0
+});
+
+export const ContextSwitchContainer = styled.div({
+    alignSelf: 'center'
+});
+
+export const DropdownButton = styled(Button)(({ theme }) => ({
+    width: 180,
+    height: 36,
+    paddingLeft: 13,
+    padding: 0,
+    border: `0px solid white`,
```

Comment:
This can just be single quoted unless you want to replace white with theme.colors.white.

Reply:
ahh.. I actually meant to make this theme.colors.white. I'll do that

Reply:
done.